docs-source/content/samples/simple/azure-kubernetes-service/_index.md

```diff
@@ -0,0 +1,975 @@
+   export t3ChannelPort="30012"
+   export exposeAdminT3Channel="false"
+   export adminNodePort="30701"
+   export exposeAdminNodePort="true"
+   export namespace="default"
+   javaOptions=-Dweblogic.StdoutDebugEnabled=false
+   export persistentVolumeClaimName="0730-azurefile-1597391432"
+   export domainPVMountPath="/shared"
+   export createDomainScriptsMountPath="/u01/weblogic"
+   export createDomainScriptName="create-domain-job.sh"
+   export createDomainFilesDir="wlst"
+   export serverPodMemoryRequest="768Mi"
+   export serverPodCpuRequest="250m"
+   export istioEnabled="false"
+   export istioReadinessPort="8888"
+
+   Generating /home/username/azure/weblogic-domains/domain1/create-domain-job.yaml
+   Generating /home/username/azure/weblogic-domains/domain1/delete-domain-job.yaml
+   Generating /home/username/azure/weblogic-domains/domain1/domain.yaml
+   Checking to see if the secret domain1-weblogic-credentials exists in namespace default
+   Checking if the persistent volume claim 0730-azurefile-1597391432 in NameSpace default exists
+   The persistent volume claim 0730-azurefile-1597391432 already exists in NameSpace default
+   W0730 07:15:52.866794   53745 helpers.go:535] --dry-run is deprecated and can be replaced with --dry-run=client.
+   configmap/domain1-create-weblogic-sample-domain-job-cm created
+   Checking the configmap domain1-create-weblogic-sample-domain-job-cm was created
+   configmap/domain1-create-weblogic-sample-domain-job-cm labeled
+   Checking if object type job with name domain1-create-weblogic-sample-domain-job exists
+   No resources found in default namespace.
+   Creating the domain by creating the job /home/weblogic/azure/weblogic-domains/domain1/create-domain-job.yaml
+   job.batch/domain1-create-weblogic-sample-domain-job created
+   Waiting for the job to complete...
+   Error from server (BadRequest): container "create-weblogic-sample-domain-job" in pod "domain1-create-weblogic-sample-domain-job-4l767" is waiting to start: PodInitializing
+   status on iteration 1 of 20
+   pod domain1-create-weblogic-sample-domain-job-4l767 status is Init:0/1
+   status on iteration 2 of 20
+   pod domain1-create-weblogic-sample-domain-job-4l767 status is Running
+   status on iteration 3 of 20
+   pod domain1-create-weblogic-sample-domain-job-4l767 status is Completed
+   domain.weblogic.oracle/domain1 created
+
+   Domain domain1 was created and will be started by the WebLogic Kubernetes Operator
+
+   Administration console access is available at http://wlswls1596-0730resourcegrou-685ba0-7434b4f5.hcp.eastus.azmk8s.io:30701/console
+   The following files were generated:
+     /home/username/azure/weblogic-domains/domain1/create-domain-inputs.yaml
+     /home/username/azure/weblogic-domains/domain1/create-domain-job.yaml
+     /home/username/azure/weblogic-domains/domain1/domain.yaml
+
+   Completed
+   ```
+
+   > **Note**: If your output does not show a successful completion, you must
```

Comment:
Maybe you could point to the Debugging suggestions in the operator documentation https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/model-in-image/debugging/ to help users debug.